pkg/security/secl/model/model.go

```diff
@@ -778,8 +778,17 @@ type ProcessCacheEntry struct {
 	refCount  uint64                     `field:"-" json:"-"`
 	onRelease func(_ *ProcessCacheEntry) `field:"-" json:"-"`
 	releaseCb func()                     `field:"-" json:"-"`
+	ProcessCacheEntrySource
 }
 
+type ProcessCacheEntrySource uint64
```

Comment:
I would put this in the ProcessContext, just calling it `Source`

Reply:
yep